hudi-cli/src/main/java/org/apache/hudi/cli/HoodieCLI.java

```diff
@@ -64,8 +64,8 @@ private static void setBasePath(String basePath) {
   }
 
   private static void setLayoutVersion(Integer layoutVersion) {
-    HoodieCLI.layoutVersion = new TimelineLayoutVersion(
-        (layoutVersion == null) ? TimelineLayoutVersion.CURR_VERSION : layoutVersion);
+    HoodieCLI.layoutVersion =
```

Comment:
For example, I am quoting this line. Do you know what rule in eclipse created this change. As checkstyle never enforced it, can we remove this rule ? Please take a look at other changes to see if there is any other pattern like this.